lib/jsdom/browser/Window.js

```diff
@@ -93,12 +94,11 @@ function Window(options) {
     }
   });
   // https://html.spec.whatwg.org/#session-history
-  this._sessionHistory = [{
+  this._sessionHistory = options.sessionHistory || new SessionHistory({
```

Comment:
This shouldn't be an option passed in, at least for now, I think. (That would require threading through api.js and the readme and such.)